### DISCUSSION TITLE
Sky update, canva mod not working

### DISCUSSION THREAD

**Discussion by Yatala6** [thatskymod/Sky-CotL-Scripts#132] [Q&A] [answered]
Hello people, first time using sky canva and first account. So recently I was using canva, but then I checked sky and then it showed to update the public game to latest version of Sky… I update it, then again I open canvas to use it but the mods don’t work anymore, for example I tried to use the speed game feature but it didn’t work, not even auto cr? Do I need to wait for the owner to make a new update to the canvas files so It can work once again?

**Comment by thatskymod** [comment, marked as answer]
Yes, that’s a common issue with Canvas, and it can be frustrating when you're just getting started. While most mods are designed to be future-proof, they still often rely on the specific version of Sky they were developed for. When a major game update is released, mod developers need time to adjust their mods to match the new game binary. This is explained further in the documentation: https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/TROUBLESHOOTING.md#updating-canvas.

However, I see that since the date of your issue, most mods have been updated and should no longer cause crashes. You can find the new updates in the [repository](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs).
